src/server/schema/resolvers/queries/getLife.ts

```diff
@@ -0,0 +1,28 @@
+import { ObjectId } from 'mongodb';
+import { getDatabaseContext } from '../../../database';
+import { GraphQLQueryResolvers, GraphQLLife } from '../definitions';
+
+const query: GraphQLQueryResolvers['getLife'] = async (root, { id }) => {
+    const { collections } = await getDatabaseContext();
+    const lifeID = new ObjectId(id);
+    const lifeCursorArray = await collections.lives.find({"_id": lifeID}).toArray();
+
+    if (lifeCursorArray.length == 0) {
+        throw new Error(`No object with ${id} found`);
+    }
+
+    const lifeFields = lifeCursorArray[0];
+    const lifeByID: GraphQLLife = {
+        firstName: lifeFields.firstName,
+        lastName: lifeFields.lastName,
+        fullName: lifeFields.firstName + " " + lifeFields.lastName,
```

Comment:
You do not need to do it here. You will need to have a type resolver to do this

Reply:
Thank you! I will work on this together with the mappers in `/starter-kit/codegen.yml`

Reply:
Also, with regards to utilising the type resolvers is this a better way to approach it?

<img width="444" alt="image" src="https://github.com/dsicol/starter-kit/assets/116002950/0560503a-1a56-4884-88ea-1ec67580c1a4">

Since the document has all fields except `fullName`, I will have to concantenate it manually in this method.

Reply:
where are you dong this? you do the concatenation in the Life Type Resolver